src/siteconfig/models.py

```diff
@@ -1,18 +1,42 @@
 from django.contrib.auth import get_user_model
 from django.core.cache import cache
 from django.core.exceptions import MultipleObjectsReturned
-from django.db import connection, models
+from django.db import connection, models, transaction
 from django.db.models.signals import post_save
 from django.dispatch import receiver
 from django.shortcuts import get_object_or_404
 from django.templatetags.static import static
+from django.conf import settings
 
 from django_tenants.utils import get_public_schema_name
 from redis import exceptions as redis_exceptions
 
 User = get_user_model()
 
 
+def get_default_deck_owner():
+    """ 
+        This is run once during site initialization
+
+        Also need to initialize owner if none exists yet.
+        (Since initialization.py loads after everything else)
+    """ 
+    qs = User.objects.filter(username=settings.TENANT_DEFAULT_OWNER_USERNAME)
+    if qs.exists():
+        return qs.first().pk
+
+    # from testing this should only run the first 3 times?
+    # Duplicates should be prevented
+    # https://stackoverflow.com/questions/21458387/transactionmanagementerror-you-cant-execute-queries-until-the-end-of-the-atom
+    with transaction.atomic():
+        owner = User.objects.create_superuser(
```

Comment:
This doesn't do what we want it to do for existing decks.  See my slack code.  Modify the logic here to make it work:
```def get_default_deck_owner():
    # handle existing decks by getting an existing staff user that is NOT bytedeck_admin:
    admin_username = settings.TENANT_DEFAULT_SUPERUSER_USERNAME
    non_admin_staff = User.objects.filter(is_staff=True).exclude(username=admin_username)

    if non_admin_staff.exists():  # i.e. there are more staff than just the admin user
        return non_admin_staff[0].id  # return the first one (assumes sorted by id, so should be the oldest one?)
    else:  # no other staff users available yet, so we'll have to create them
        deck_owner = User.objects.create(username=settings.TENANT_DEFAULT_DECK_OWNER_USERNAME, is_staff=True)
        return deck_owner.id
```

Reply:
done